src/entities/RfqmQuoteEntity.ts

```diff
@@ -0,0 +1,52 @@
+import { RfqOrder } from '@0x/asset-swapper';
+import { Column, Entity, Index, PrimaryColumn } from 'typeorm';
+
+import { RfqmFee } from '../services/rfqm_service';
+
+@Entity({ name: 'rfqm_quotes' })
+export class RfqmQuoteEntity {
+    @PrimaryColumn({ name: 'order_hash', type: 'varchar' })
+    public orderHash?: string;
```

Comment:
```suggestion
    public orderHash: string;
```

Reply:
This one is a bit funny - by some oddity of how our `ormconfig.ts` works, it really wants this to be `orderHash?`. There's probably a way to make it just be `orderHash`, but not sure if it's worth it at this moment. Will look into this if I have some more time